.github/workflows/build_test.yml

```diff
@@ -10,25 +10,21 @@ jobs:
       matrix:
         ros_distribution:
           - foxy
-          - galactic
+          - humble
           - rolling
         include:
           # Foxy Fitzroy (May 2020 - May 2023)
           - docker_image: rostooling/setup-ros-docker:ubuntu-focal-ros-foxy-ros-base-latest
             ros_distribution: foxy
             ros_version: 2
-          # Galactic Geochelone (May 2021 - November 2022)
-          - docker_image: rostooling/setup-ros-docker:ubuntu-focal-ros-galactic-ros-base-latest
-            ros_distribution: galactic
+          # Humble Hawksbill (May 2022 - May 2027)
+          - docker_image: rostooling/setup-ros-docker:ubuntu-jammy-ros-humble-ros-base-latest
+            ros_distribution: humble
             ros_version: 2
           # Rolling Ridley  (June 2020 - Present)
-          - docker_image: rostooling/setup-ros-docker:ubuntu-focal-ros-rolling-ros-base-latest
+          - docker_image: rostooling/setup-ros-docker:ubuntu-jammy-ros-rolling-ros-base-latest
             ros_distribution: rolling
             ros_version: 2
-        linter:
```

Comment:
Any reason for removing the lint checks?

Reply:
I think linters should be passed to `action-ros-lint` like [this](https://github.com/ros-tooling/topic_tools/blob/fdc8df8361fd449c4e6564a4fdb04ecbd6771870/.github/workflows/lint.yml#L28-L30), but currently they are never used, so the same ci is executed 3 times wrongly.
https://github.com/ros-drivers/transport_drivers/actions/runs/4049784318/jobs/7144340139
https://github.com/ros-drivers/transport_drivers/actions/runs/4049784318/jobs/7144340221
https://github.com/ros-drivers/transport_drivers/actions/runs/4049784318/jobs/7144340325

Reply:
@esteve friendly ping

Reply:
@wep21 sorry, was away for a few days. Thanks for the explanation